akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -293,11 +372,12 @@ final class ClusterClient(settings: ClusterClientSettings) extends Actor with Ac
           contacts = contactPoints.map(context.actorSelection)
```

Comment:
use a separate field to store the `ActorPath`, 

```
contactPaths = contactPoints.map(ActorPaths.fromString)
contacts = contactPaths.map(context.actorSelection)
```

yeah, it requires mainting two fields, but the purpose is different, going from `ActorSelection` to `ActorPath` via the `anchorPath` is not good, I think it just happens to work here and not something that can be done with actor selections in general